tools/openapi-plugin/README.md

```diff
@@ -16,11 +16,16 @@ To build a new client in the clients folder:
 To build with flags (without needing to prompt):
 Run `nx g client` with optional flags:
 
-- --name Name of the api client.
-- --schemaPath Path to the schema. If using the --remote flag then you must specify a valid remote URL. If not you must specify a local file.
-- --configPath path to the redocly config file responsible for auth. For more information: https://openapi-ts.dev/cli#auth.
-- --remote Specify whether you would like to fetch remotely.
+- `--validate` **Recommended**. If passed, this will pre-validate the schema file and only generate the client if the given file is a valid schema. 
+- `--name` Name of the api client.
+- `--schemaPath` Path to the schema. If using the --remote flag then you must specify a valid remote URL. If not you must specify a local file.
+- `--remote` Specify whether you would like to fetch remotely.
+- `--configPath` path to the redocly config file responsible for authentication when fetching a schema remotely. For more information: https://openapi-ts.dev/cli#auth.
 
+
+### Regenerating types for an existing client
+
+To regenerate an existing client run the same generator command again (as above). If the client already exists in the Nx project, after confirmation it will have its types file regenerated from either the provided. 
```

Comment:
Is this sentence complete? I feel like it need something more after `either the provided`.